react/features/external-api/middleware.ts

```diff
@@ -124,6 +125,10 @@ MiddlewareRegistry.register(store => next => action => {
         break;
     }
 
+    case DATA_CHANNEL_CLOSED:
+        APP.API.notifyDataChannelClosed(action.code, action.reason);
```

Comment:
The function definition and mapping of the event name in the external API is missing.

Reply:
Opsie! I'll fix this tomorrow, thanks Jaya!